### PR TITLE
sumologicexporter: add support for all types of log body

### PR DESCRIPTION
**Pull request by sumo-drosiek**
**Description:**

- Add support for all types of log body (only string has been supported so far)
- Refactor fields (change type to AttributeMap) to simplify above change

**Link to tracking Issue:** N/A

**Testing:** Unit-tests, manual tests

**Documentation:** N/A

### CODE REVIEW

**Comment by sumo-drosiek** [comment]
cc: @kkujawa-sumo @pmalek-sumo

**Review by kasia-kujawa — APPROVED** [review]

**Review by pmalek-sumo — APPROVED** [review]

**Comment by tigrannajaryan** [comment]
@pmm-sumo please review as the codeowner.

File: exporter/sumologicexporter/fields.go [hunk 1]
```diff
@@ -18,18 +18,27 @@ import (
...
 // string returns fields as ordered key=value string with `, ` as separator
 func (f fields) string() string {
-	rv := make([]string, 0, len(f))
-	for k, v := range f {
-		rv = append(rv, fmt.Sprintf("%s=%s", k, v))
-	}
-	sort.Strings(rv)
+	returnValue := make([]string, 0, f.orig.Len())
+	f.orig.ForEach(func(k string, v pdata.AttributeValue) {
+		returnValue = append(returnValue, fmt.Sprintf("%s=%s", k, tracetranslator.AttributeValueToString(v, false)))
```
Comment (pmm-sumo) [inline exporter/sumologicexporter/fields.go:39]: Does it handle escaping and such?

Comment (sumo-drosiek) [reply exporter/sumologicexporter/fields.go]: Good point, I added sanitization for `,`, `=` and `\n`

**Review by pmm-sumo — APPROVED** [review]

**Review by tigrannajaryan — APPROVED** [review]

**Pull request merged**